contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -155,12 +171,14 @@ def _get_workflow_tasks(self, exec_id, last_query_time=None):
         result = []
 
         try:
-            query_filters = {}
+            wf_tasks = self._client.tasks.list(workflow_execution_id=exec_id)
```

Comment:
Not very expensive. It's still filtering by the execution ID which limits the result set. I had to remove the filter because there's an edge case where the query filter doesn't work.